website/templates/emails/registration_bulk_upload_success_partial.html.mako

```diff
@@ -0,0 +1,33 @@
+<%inherit file="notify_base.mako" />
+<%def name="content()">
+<tr>
+  <td style="border-collapse: collapse;">
+    <h3 class="text-center" style="padding: 0;margin: 0;border: none;list-style: none;font-weight: 300;text-align: center;"> Some Registrations Successfully Bulk Uploaded to your Community's Registry</h3>
+  </td>
+</tr>
+<tr>
+  <td style="border-collapse: collapse;">
+      Hello ${fullname},<br>
+      <br>
+      % if auto_approval:
+          ${successes} out of ${total} of your registrations were successfully uploaded! Click the linkbutton below to begin moderating their recently submitted registrations.<br>
```

Comment:
"linkbutton"?

Reply:
Good catch, fixed in https://github.com/CenterForOpenScience/osf.io/pull/9783